Libraries/Image/RCTImageLoader.m

```diff
@@ -390,31 +401,6 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
       request = mutableRequest;
     }
 
-    // Check for cached response before reloading
-    // TODO: move URL cache out of RCTImageLoader into its own module
-    NSCachedURLResponse *cachedResponse = [_URLCache cachedResponseForRequest:request];
-
-    while (cachedResponse) {
-      if ([cachedResponse.response isKindOfClass:[NSHTTPURLResponse class]]) {
-        NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *)cachedResponse.response;
-        if (httpResponse.statusCode == 301 || httpResponse.statusCode == 302) {
```

Comment:
cc'ing original author @hayeah 

We don't implement URLSession:willPerformHTTPRedirection: in RCTHTTPRequestHandler. I'm not sure what the default behaviour is here.